renderer/js/mtftemplate.js

```diff
@@ -1,42 +1,209 @@
+    <% _.each(MTFController.selAns,function(val,key){ %>\
+      <div class='mtf-vert-option <%= MTFController.optionsHeight %>'>\
+        <div class='mtf-vert-ques-option'>\
+          <div class='mtf-vert-ques-text'>\
+            <div class='mtf-vert-ques-text-inner cont-dragula' id='left<%= (key+1) %>' leftindex='<%= val.index %>'><% if(val.selText.length > 0){ %> <p><%= val.selText  %> </p> <% }else{ %><%= val.selText %><% } %></div>\
+          </div>\
+        </div>\
+      </div>\
+    <% });%>\
+  </div>\
+  <div class='mtf-vert-container panel panel-body'>\
+    <% _.each(question.data.option.optionsRHS,function(val,key){ %>\
+      <div class='mtf-vert-option <%= MTFController.optionsHeight %>'>\
+        <div class='mtf-vert-ques-option'>\
+          <div class='mtf-vert-ques-text'>\
+            <div class='mtf-vert-ques-text-inner cont-dragula' id='right<%= (key+1) %>' mapIndex='<%= val.mapIndex %>'><% if(MTFController.selAns[key].selText < 1){ %>\
+              <p onclick='MTFController.showImageModel(event, \"background-image\")' style=\"background-size:100% 100%; background-image:url('<%= val.image %>');\">\
+                <% if(val.audio){ %> \
+                  <span class='mtf-vert-opt-audio-image' >\
+                    <img  src='<%=MTFController.pluginInstance.getAudioIcon('renderer/assets/audio.png') %>' onclick=MTFController.pluginInstance.playAudio({src:'<%= val.audio %>'}) \>\
+                  </span>\
+                <% } %>\
+                <%= val.text %>\
+              </p> \
+              <% } %>\
+            </div>\
+          </div>\
+        </div>\
+      </div>\
+    <% });%>\
+  </div>";
+}
+
+/**
+ * image will be shown in popup
+ * @memberof org.ekstep.questionunit.mtf.mtftemplate
+ */
+MTFController.showImageModel = function (event, imageSrcType) {
+  var eventData;
+  if (imageSrcType == "background-image") {
+    eventData = event.target.style.backgroundImage.slice(4, -1).replace(/"/g, "");
+    if (!eventData) return false; //when there is no image option
+  } else {
+    eventData = event.target.src;
+  }
+  var modelTemplate = "<div class='popup' id='image-model-popup' onclick='MTFController.hideImageModel()'><div class='popup-overlay' onclick='MTFController.hideImageModel()'></div> \
+  <div class='popup-full-body'> \
+  <div class='font-lato assess-popup assess-goodjob-popup'> \
+    <img class='qc-question-fullimage' src=<%= src %> /> \
+    <div onclick='MTFController.hideImageModel()' class='qc-popup-close-button'>X</div> \
+    <div  class='qc-popup-close-button'>X</div> \
+  </div></div>";
+  var template = _.template(modelTemplate);
+  var templateData = template({
+    src: eventData
+  })
+  $(MTFController.constant.qsMTFElement).append(templateData);
+},
+
+/**
+ * onclick overlay or X button the popup will be hide
+ * @memberof org.ekstep.questionunit.mtf.mtftemplate
+ */
+MTFController.hideImageModel = function () {
+  $("#image-model-popup").remove();
```

Comment:
Handle through event parameter instead of id.
MTFController.hideImageModel(event)